scripts/module_overview/module_overview.py

```diff
@@ -220,6 +221,42 @@ def generate_general_overview() -> None:
     print(f"Module overview created at {md_fn}")
 
 
+def generate_overview_json_data(modules: dict) -> dict:
+    """
+    Generate the data for the json overview.
+
+    @param modules: Dict with all the modules per cluster. Keys are the cluster names.
+    @return: Dictionary with the required JSON structure.
+    """
+    json_data = {"clusters": list(modules.keys()), "modules": {}}
+    avail_software = get_unique_software_names(modules)
+    all_software = get_unique_software_names(np.concatenate(list(avail_software.values())))
+
+    # creates a list of booleans for each software that indicates
+    # if the software is available for the corresponding cluster.
+    for soft in all_software:
+        available = []
+        for cluster in json_data["clusters"]:
+            available.append(soft in avail_software[cluster])
```

Comment:
Plan was to use `0` and `1` to make the JSON smaller?

```suggestion
            # use 0 and 1 rather than true/false to obtain smaller JSON file
            available.append(int(soft in avail_software[cluster]))

Reply:
Yes, this is already implemented in #559